vrk-kpa/suomidigi
# SUOPA-796 User profile linking

To test: 
- `make fresh`
   - Check that the `drush updb` has been run. There should be updates for the user profiles.
- Log in as an admin and with a different browser (or incognito) create a new user https://suomidigi.docker.sh/user/register
   - The local environment won't send an email, so it will give an error when trying to create user account. 
   - Take a note of the email address you've typed in for the account, run `drush uli --mail=the-email@you-enter.ed` and open the link in the browser you've used to create the user account
- Once you've logged in with the new user, give it some details f.e. Title, Organisation, Picture and Description and click on save. **1)**
   - Go to https://suomidigi.docker.sh/user and make sure the user has the information you've entered
   - Go to https://suomidigi.docker.sh/node/add/event and fill in the fields and save the event
   - Go back to your user account and check that the event has appeared to the My content list

- Switch to the browser where admin is logged in and go to https://suomidigi.docker.sh/en/admin/people
   - Search for the user you've created and edit the user account. 
   - Give the user `content producer` role and save the user
- Switch back to the other browser and go to https://suomidigi.docker.sh/node/add/blog_post
   - Check that the writer term has been automatically added to the node 
![Luo Blogikirjoitus | Suomidigi 2020-02-25 12-54-09](https://user-images.githubusercontent.com/1712902/75241012-fe06eb80-57cd-11ea-93e5-5d3430513bed.jpg)
  - Fill in all of the required fields and save the node. 
  - Check that the Author has the details you've entered to the user account **1)**
  - Finally go to your profile page (https://suomdigi.docker.sh/user) and check that the blog post has been added to the "My content" list as well.
